doc/src/base/sort.md

```diff
@@ -171,12 +171,12 @@ By default, `sort`, `searchsorted`, and related functions use [`isless`](@ref) t
 two elements in order to determine which should come first. The
 [`Base.Order.Ordering`](@ref) abstract type provides a mechanism for defining alternate
 orderings on the same set of elements. Instances of `Ordering` define a
-[strict partial order](https://en.wikipedia.org/wiki/Partially_ordered_set#Strict_partial_order).
-To be a strict partial order, for any elements `a`, `b`, `c` the following hold:
+[strict weak order](https://en.wikipedia.org/wiki/Weak_ordering#Strict_weak_orderings).
+To be a strict weak order, for any elements `a`, `b`, `c` the following hold:
 
-* if `a == b`, then `lt(a, b) == false`;
-* `lt(a, b) && lt(b, a) == false`; and
-* if `lt(a, b) && lt(b, c) == true`, then `lt(a, c) == true`
+* `lt(a, b) && lt(b, a) === false`;
+* if `lt(a, b) && lt(b, c)`, then `lt(a, c)`; and
+* if `!lt(a, b) && !lt(b, c)`, then `!lt(a, c)`
```

Comment:
As discussed in https://github.com/JuliaLang/julia/pull/48387 it might be good to add the irreflexivity condition even if it's implied by the asymmetric condition...